apps/web/app/hooks/features/useTimesheet.ts

```
@@ -2,148 +2,163 @@
+	(items: TimesheetLog[]): GroupedTimesheet[] => {
+		if (!items?.length) return [];
+		const groupedByDate = items.reduce(
+			(acc, item) => {
+				if (!item?.timesheet?.createdAt) {
+					console.warn('Skipping item with missing createdAt:', item);
+					return acc;
+				}
+
+				try {
+					const date = new Date(item.timesheet.createdAt);
+					if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)